content/millennium/r4/medications/immunization.md

```diff
@@ -110,6 +123,41 @@ List an individual Immunization by its id:
 <%= headers status: 200 %>
 <%= json(:R4_IMMUNIZATION_ENTRY) %>
 
+Note: The examples provided here are non-normative and replaying them in the public sandbox is not guaranteed to yield the results shown on the site.
+
+#### Patient Authorization Request For Completed Status
+
+    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Immunization/M197197986
+
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:R4_IMMUNIZATION_PATIENT_ENTRY) %>
+
+Note: The examples provided here are non-normative and replaying them in the public sandbox is not guaranteed to yield the results shown on the site.
+
+#### Patient Authorization Request For Entered in Error Status
+
+    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Immunization/M211860902
+
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:R4_IMMUNIZATION_ENTERED_IN_ERROR_STATUS) %>
+
+Note: The examples provided here are non-normative and replaying them in the public sandbox is not guaranteed to yield the results shown on the site.
+
+#### Patient Authorization Request For Not done Status
+
+    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Immunization/HM178147794
```

Comment:
Update url to https://fhir-ehr.cerner.com

Reply:
Addressed in commit ID df37656